helm/volsync/values.yaml

```diff
@@ -33,6 +33,8 @@ syncthing:
   # Overrides the image tag whose default is the chart appVersion.
   tag: ""
   image: ""
+kube-rbac-proxy:
+  image: quay.io/brancz/kube-rbac-proxy:v0.18.1
```

Comment:
Can we follow the `repository`, `tag`, `image` pattern that we use for the other images or is there something special about the rbac proxy?

Reply:
I guess I thought it would simply make the renovate replacement easier, as we would normally replace the entire image path here: https://github.com/backube/volsync/blob/main/config/default/manager_auth_proxy_patch.yaml#L35

If we ever didn't set tag, it also means we'd default to the volsync appVersion, but that probably isn't a real concern.

But...we could also do:

```
kube-rbac-proxy:
  repository: quay.io/brancz/kube-rbac-proxy
  tag: v0.18.1
  image: ""
```

Maybe the best question is:  Can renovate automatically update this and which format is easiest?  I didn't see a specific rule for kube-rbac-proxy but we do have something for docker images in yamls...which maybe would work on the values.yaml as well?

https://github.com/backube/volsync/blob/main/renovate.json5#L152-L161

Reply:
We can customize renovate to handle the separate tag field.

Reply:
Gotcha, will update to use repository & tags then 👍